docs/deployments/aws/aws-s3-create-bucket/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Amazon S3 Create Bucket
+description: Create a new bucket in Amazon S3.
+---
+
+Octopus supports creating a new S3 bucket through the `Create an Amazon S3 Bucket` step.
+This step provides a way to create or manage an arbitrary Amazon S3 Bucket. This could be especially useful in conjunction with AWS Lambda functions.
+
+:::hint
+The `Create an Amazon S3 Bucket` manages the S3 bucket via CloudFormation.
```

Comment:
Because this step was delivered via the step package feed it is actually available in both 2022.1 and 2022.2. @tekerson was there a minimum build number besides having the feed enabled?

Reply:
@rhysparry It should be available in any build that has the feed enabled. I don't know off the top of my head when it was enabled. Do you have that build number? should it be mentioned here?

Reply:
So that will be 2021.1.1933, which was before the first GA 2021.1 build, so no need for the build number!